dev/BIDS_to_qmrlab_input_mappings.json

```diff
@@ -0,0 +1,112 @@
+{
```

Comment:
Note: qMRLab BIDS files currently in a temporary `dev/` directory, which will need to be renamed (and possible moved) prior to merging.

Reply:
The `dev/` directory was not meant to be temporary, but we can rename it to `config` if you think that it makes more sense.